### PR TITLE
[#175] Container now accepts Makefile parameters

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ Running the docker image will save artifacts into `$(pwd)/build`:
 $ docker run --rm -v $(pwd):/irods_docs:ro -v $(pwd)/build:/hostcomputer irods_docs_builder
 ```
 
+You can generate documentation for a specific build of iRODS by running the following:
+
+```
+$ docker run --rm -v $(pwd):/irods_docs:ro -v $(pwd)/build:/hostcomputer irods_docs_builder MAKEGITHUBACCOUNT=<account> MAKEIRODSVERSION=<branch-or-commit>
```
Comment (trel) [inline README.md:38]: take it or leave it...  consider...  MAKEIRODSVERSION -> MAKEGITCOMMITTISH ?

Comment (korydraughn) [reply README.md:38]: How do you feel about removing the `MAKE` prefix and adding underscores?
- `GITHUB_ACCOUNT`
- `GIT_COMMITTISH`

Comment (trel) [reply README.md:38]: agreed, those seem better all around.

Comment (korydraughn) [reply README.md]: done

**Review by trel — APPROVED** [review]
looks good by inspection.   please # when satisfied this is functional/good.

**Comment by korydraughn** [comment]
Added #

File: Makefile [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 SHELL = /bin/bash
 
-MAKEGITHUBACCOUNT = irods
-MAKEIRODSVERSION = main
+GITHUB_ACCOUNT = irods
+GIT_COMMITTISH = main
```
Comment (alanking) [inline Makefile:6]: Should this be `GIT_COMMITISH`? Never seen it with two t's before.

**Review by alanking — COMMENTED** [review]
Just one question regarding spelling of one of the variables

File: Makefile (hunk 2 above)
Comment (alanking) [reply Makefile:6]: Oops, should have looked it up and this is the way to go. Looks good

https://git-scm.com/docs/gitglossary

**Pull request merged**